jasstests/jassrealtime/search/test_document.py

```diff
@@ -445,6 +446,84 @@ def test_get_annotations_by_document_one_type(self):
         tokens = ds.get_annotations_for_one_type("bucket1", "token")
         self.assertEqual(len(tokens["corpus1"]["bucket1"]["token"]), 6)
 
+    def test_count_annotations_for_types(self):
+        global envIdReadOnly
+        global authorizationReadOnly
```

Comment:
I would prefer using the test class members instead of using globals. (Personal preference).